### DISCUSSION TITLE
Error Adding Custom Fields in it

### DISCUSSION THREAD

**Discussion by aadeshbhujbal** [pwndoc/pwndoc#537] [Q&A]
Hi all ,
i am Trying to add Custom Fields cleanup2 to it but it is giving undefinded cannot read properties of undefined reading substring error

![image](https://github.com/user-attachments/assets/204ca1ac-dbbc-4f2d-9158-1062e7ad2f6c)
![image](https://github.com/user-attachments/assets/09891f15-6043-4786-8302-b9f5682e8770)
![image](https://github.com/user-attachments/assets/81e2a9be-09af-4a34-9f2b-cd7f5f4dc45b)

how to add custom field in docs??

**Comment by yeln4ts** [comment]
Your custom field is an "Input" but you used the notation in your docx for a "Text" field.
Just use {cleanup2}
